src/common/base.search.ts

```diff
@@ -0,0 +1,14 @@
+import { ISearchInput } from "./interfaces/search.interface"
+
+const getQuery = (searchCriteria?: ISearchInput) => {
+  const query: any = {}
+  const limit = searchCriteria?.limit ?? 10
+  const skip = searchCriteria?.skip ?? 0
+
+  query.limit = limit
+  query.offset = (limit*skip)
+
+  return query
+}
+
+export { getQuery }
```

Comment:
What are your thoughts (if any) on ending files in a new line?

Reply:
it's a good approach to know the end of the file, to be honest, didn't remember this